### PR TITLE
11th Translation - 1088 Words

### PR DESCRIPTION
**Pull request by cherryloua**
Hi everyone and @severinolorillajr 

Please, take some time to review or merge my PR. Thanks.

### CODE REVIEW

File: book/08-customizing-git/sections/hooks.asc [hunk 1]
```diff
@@ -15,95 +15,98 @@ Kung ikaw ay nagpasimula ng bagong repositoryo sa pamamagitan ng `git init`, ang
...
 
-To enable a hook script, put a file in the `hooks` subdirectory of your .git directory that is named appropriately (without any extension) and is executable.
-From that point forward, it should be called.
-We'll cover most of the major hook filenames here.
+Para paganahin ang script na hook, maglagay ng file sa `hooks` na subdirectory sa iyong .git na direktoryo na nakapangalan ng maayos (na walang kahit na anong ekstensiyon) at ito ay maipapatupad.
+Mula sa puntong iyon, ito ay dapat tinatawag.
+Tatalakayin namin ang karamihan sa mga pangunahing hook na mga filename dito.
 
-==== Client-Side Hooks
+==== Mga Hook na Kliyenteng Panig
```
Comment (fatelrou) [inline book/08-customizing-git/sections/hooks.asc:23]: @severinolorillajr should we translate Client-Side to Kliyenteng Panig?

File: book/08-customizing-git/sections/hooks.asc [hunk 2]
```diff
@@ -15,95 +15,98 @@ Kung ikaw ay nagpasimula ng bagong repositoryo sa pamamagitan ng `git init`, ang
...
-If your intent with these scripts is to enforce a policy, you'll probably want to do that on the server side; see the example in <<_customizing_git#_an_example_git_enforced_policy>>.
+Ito ay mahalagang tandaan na ang mga hook na kliyenteng panig ay *hindi* kinokopya kapag ikaw ay nag-clone ng isang repositoryo.
+Kung ang iyong ninanais ay ipapatupad ang isang patakaran sa pamamagitan ng mga script na ito, marahil gusto mong gawin ito sa bahagi ng server; tingnan ang halimbawa sa <<_customizing_git#_an_example_git_enforced_policy>>.
 ====
 
-===== Committing-Workflow Hooks
+===== Mga Hook na Committing-Workflow
 
-The first four hooks have to do with the committing process.
+Ang unang apat na mga hook ay may kinalaman sa pag-commit ng proseso.
```
Comment (fatelrou) [inline book/08-customizing-git/sections/hooks.asc]: "pag-commit ng proseso" change to "pag-commit na proseso" o "prosesong pag-commit"

File: book/08-customizing-git/sections/hooks.asc [hunk 3]
```diff
@@ -15,95 +15,98 @@ Kung ikaw ay nagpasimula ng bagong repositoryo sa pamamagitan ng `git init`, ang
...
+Maaari mong gawin ang mga bagay kagaya ng pagsuri sa estilo ng code (magpatakbo ng `lint` o katumbas nito), magsusuri ng whitespace sa hulihan (ang default na hook ay eksaktong gumagawa nito), o suriin ang angkop na dokumentasyon sa mga bagong paraan.
 
-The `prepare-commit-msg` hook is run before the commit message editor is fired up but after the default message is created.
-It lets you edit the default message before the commit author sees it.
-This hook takes a few parameters: the path to the file that holds the commit message so far, the type of commit, and the commit SHA-1 if this is an amended commit.
-This hook generally isn't useful for normal commits; rather, it's good for commits where the default message is auto-generated, such as templated commit messages, merge commits, squashed commits, and amended commits.
-You may use it in conjunction with a commit template to programmatically insert information.
+Ang `prepare-commit-msg` na hook ay pinatakbo bago ang editor ng mensahe ng commit ay napatakbo ngunit pagkatpos na nagawa ang default na mensahe.
```
Comment (fatelrou) [inline book/08-customizing-git/sections/hooks.asc]: "ngunit pagkatpos" typo, change it to "ngunit pagkatapos"

File: book/08-customizing-git/sections/hooks.asc [hunk 4]
```diff
@@ -15,95 +15,98 @@ Kung ikaw ay nagpasimula ng bagong repositoryo sa pamamagitan ng `git init`, ang
...
-In the last section of this chapter, We'll demonstrate using this hook to check that your commit message is conformant to a required pattern.
+Ang hook na ito ay hindi nakakatulong sa pangkalahatan para sa normal na commit; sa halip, ito ay maganda para sa mga commit kung saan ang default na mensahe ay awtomatikong nabuo, kagaya ng mga mensaheng naka-template na commit, mga merge na commit, na-squash na mga commit, at mga commit na nabago.
 
-After the entire commit process is completed, the `post-commit` hook runs.
-It doesn't take any parameters, but you can easily get the last commit by running `git log -1 HEAD`.
-Generally, this script is used for notification or something similar.
+Maaari mong gamitin ito kasabay ang template na commit para naka-programa ang pagpasok sa impormasyon.
+
+Ang `commit-msg` na hook ay kumukuha ng isang sadyansukat, kung saan ito ulit ang path ng pansamantalang file na nilalaman ang mensahe ng commit na sinulat ng developer.
+Kung ang script na ito ay lalabas sa di-sero, ang Git ay bibigo sa proseso ng commit, kaya maaari mong gamitin ito para patunayan ang stado ng iyong proyekto o mensahe ng commit bago pahintulotan makadaan ang isang commit.
```
Comment (fatelrou) [inline book/08-customizing-git/sections/hooks.asc]: "patunayan ang stado" typo again, change it to "estado"

File: book/08-customizing-git/sections/hooks.asc [hunk 5]
```diff
@@ -15,95 +15,98 @@ Kung ikaw ay nagpasimula ng bagong repositoryo sa pamamagitan ng `git init`, ang
...
-It receives the name and location of the remote as parameters, and a list of to-be-updated refs through `stdin`.
-You can use it to validate a set of ref updates before a push occurs (a non-zero exit code will abort the push).
+Ang `pre-push` na hook ay tumatakbo sa panahon ng `git push`, pagkatops nabago ang remote na refs ngunit bago paman na lipat ang kahit na anong bagay.
+Ito ay nakakatanggap ng pangalan at lokasyon ng remote bilang mga sadyansukat, at isang listahan ng mga ibabagong mga ref sa pamamagitan ng `stdin`.
+Maari mong gamitin ito para patunayan ang mga pagbabago sa isang pangkat ng ref bago paman ang isang push ay maisagawa (ang isang di-sero na panlabas na code ay magpapabigo sa push).
 
-Git occasionally does garbage collection as part of its normal operation, by invoking `git gc --auto`.
-The `pre-auto-gc` hook is invoked just before the garbage collection takes place, and can be used to notify you that this is happening, or to abort the collection if now isn't a good time.
+Paminsa-minsan ang Git ay nagsasagaw ng koleksyon ng basura bilang bahagi ng kanyang normal na operasyun, sa pamamagitan ng pagtawag sa `git gc --auto`.
```
Comment (fatelrou) [inline book/08-customizing-git/sections/hooks.asc]: "nagsasagaw ng koleksyon" typo, "nagsasagawa ng koleksyon"

File: book/08-customizing-git/sections/hooks.asc [hunk 6]
```diff
@@ -15,95 +15,98 @@ Kung ikaw ay nagpasimula ng bagong repositoryo sa pamamagitan ng `git init`, ang
...
-You can use it to restore data in the working tree that Git can't track, such as permissions data.
-This hook can likewise validate the presence of files external to Git control that you may want copied in when the working tree changes.
+Ang `post-merge` na hook ay tatakbo pagkatapos ng isang matagumpay ang `merge` na utos.
+Maaari mong gamitin ito para ibalik ang data sa gumaganang tree na hindi masubaybayan ng Git, kagaya ng mga pahintulot na data.
+Ang hook na ito ay maaaring magpapatunay din sa pagkakaroon ng mga file na panlabas na pagkontrol ng Git na maaaring gusto mong kopyahin kung ang gumaganang tree ay nagpapalit.
 
-The `pre-push` hook runs during `git push`, after the remote refs have been updated but before any objects have been transferred.
-It receives the name and location of the remote as parameters, and a list of to-be-updated refs through `stdin`.
-You can use it to validate a set of ref updates before a push occurs (a non-zero exit code will abort the push).
+Ang `pre-push` na hook ay tumatakbo sa panahon ng `git push`, pagkatops nabago ang remote na refs ngunit bago paman na lipat ang kahit na anong bagay.
```
Comment (fatelrou) [inline book/08-customizing-git/sections/hooks.asc]: "remote na refs" change it to "remote na mga ref"

File: book/08-customizing-git/sections/hooks.asc [hunk 7]
```diff
@@ -15,95 +15,98 @@ Kung ikaw ay nagpasimula ng bagong repositoryo sa pamamagitan ng `git init`, ang
...
 
-After you run a successful `git checkout`, the `post-checkout` hook runs; you can use it to set up your working directory properly for your project environment.
-This may mean moving in large binary files that you don't want source controlled, auto-generating documentation, or something along those lines.
+Pagkatapos mong magpatakbo ng isang matagumpay na `git checkout`, ang `post-checkout` na hook ay tatakbo; maari mong gamitin ito para i-setup ang iyong gumaganang direktoryo ng maaayos para sa environment ng iyong proyekto.
+Ito ay maaring mangahulugan sa paglipat ng malaking binary na mga file na hindi mo gustong i-kontrol ang pinanggagalingan, awtomatikong paglikha ng dokumento, o isang bagay sa mga linyang iyon.
 
 The `post-merge` hook runs after a successful `merge` command.
-You can use it to restore data in the working tree that Git can't track, such as permissions data.
-This hook can likewise validate the presence of files external to Git control that you may want copied in when the working tree changes.
+Ang `post-merge` na hook ay tatakbo pagkatapos ng isang matagumpay ang `merge` na utos.
```
Comment (fatelrou) [inline book/08-customizing-git/sections/hooks.asc]: "matagumpay ang `merge` na utos" change it to "matagumpay na `merge` na utos"

**Review by fatelrou — COMMENTED** [review]
Hi @cherryloua 

A few suggestions here, mostly are typos.

**Comment by cherryloua** [comment]
Hi @fatelrou thank you for your review. 
The suggested changes are not applied in my latest commit.
@severinolorillajr you may now check or merge the PR. Thanks!

**Pull request merged**